src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -57,11 +57,16 @@
     <BuildArgs>$(BuildArgs) -bl</BuildArgs>
     <BuildArgs>$(BuildArgs) /p:CopySrcInsteadOfClone=true</BuildArgs>
     <BuildArgs Condition="'$(CrossBuild)' == 'true'">$(BuildArgs) /p:CrossBuild=$(CrossBuild)</BuildArgs>
-    <BuildArgs Condition="'$(DotNetBuildVertical)' == 'true'">$(BuildArgs) /p:ArcadeBuildVertical=true</BuildArgs>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(DotNetBuildVertical)' == 'true'">
+    <BuildArgs>$(BuildArgs) /p:ArcadeBuildVertical=true</BuildArgs>
+    <BuildArgs>$(BuildArgs) /p:DotNetBuildRepo=true</BuildArgs>
```

Comment:
I don't have the new controls in memory anymore but if I understand this correctly we now pass `DotNetBuildRepo` in, regardless of only building from source or not. Right?

Reply:
Correct. it represents the baseline VMR product style builds. I'll hoist it.